spigot-tests/tests/acd-screen-share-test.js

```diff
@@ -105,17 +105,19 @@ describe('ACD Screen Share via webrtc-sdk [acd-screenshare-sdk] [sdk] [stable]',
     await testUtils.timeout(4000);
 
     const peerStreamAdded = new Promise((resolve, reject) => {
-      setTimeout(() => reject(new Error('Timeout waiting for remote stream')), config.validationTimeout);
+      const timer = setTimeout(() => reject(new Error('Timeout waiting for remote stream')), config.validationTimeout);
```

Comment:
I wonder if the spigot `rejectTimeout()` util would work for this use case? We wouldn't need to manage timers if it did work. If it doesn't work, then we could try to change the util to support our use case.